decentraland/unity-renderer
# chore: passport performance improvements

## What does this PR change?

Improves passport performances in two key areas described in issue #4539 

...

## How to test the changes?



1. Launch the explorer https://play.decentraland.org/?explorer-branch=chore/passport-improvements
2. Use the passport and check it works as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
